core/engine/src/tests/operators.rs

```diff
@@ -600,6 +600,15 @@ fn delete_in_strict_function_returned() {
     )]);
 }
 
+#[test]
+fn regex_slash_eq() {
+    run_test_actions([
+        TestAction::assert_eq("+/=/", JsValue::nan()),
+        TestAction::assert_eq("var a = 5; /=/; a", 5),
+        TestAction::assert_eq("x = () => /=/;\n\"a=b\".match(x())[0]", js_str!("=")),
```

Comment:
Can you expand on the `\n` here`? Is there a specific reason for it?
When I try to run `x = () => /=/;"a=b".match(x())[0]` on the current main branch I get a syntax error, in node I get `=` as expected.

Reply:
The syntax error is not expected here:
`/=/` is a valid regex;
so `x = () => /=/` is also valid as arrow function that returns mentioned regex;
so `x = () => /=/;"a=b".match(x())[0]` should to create arrow fn `x` that return regex and then call for `"a=b"` match function with `x()` arg that will be the same as `"a=b".match(/=/)` that return `["="]` and `[0]` for it will return `"="`.
So `x = () => /=/;"a=b".match(x())[0]` also should return `"="`

Reply:
Yeah that's all clear, I was just wondering about the line break `\n`.

Reply:
Oh sorry, I misunderstood your question.

No, there is no special reason.
The test have such form because firstly I checked bug in js file, and this was on two diff lines so seems like I just moved new line with the test from js.